.github/workflows/nodejs.yml

```diff
@@ -29,5 +29,8 @@ jobs:
       - name: Run unit tests
         run: yarn test
 
+      - name: Check if JSON conversions changed by running tests
+        run: git diff --exit-code "src/data/*.json"
```

Comment:
I verified this change to the action here: https://github.com/OpenBeta/sandbag/actions/runs/5406330482/jobs/9823085987